<!--xml-->
<launch>
    <node name="$(env ROS_HOSTNAME)_master_node" pkg="icarus_rover_v2" type="master_node"  output="screen" clear_params="true">
	    <param name="verbosity_level"       value="DEBUG"/>
        <param name="require_pps_to_start"  value="false"/>    
        <param name="loop1_rate"            value="1"/> <!-- device,resource available, -->    
    </node>
    <node name="$(env ROS_HOSTNAME)_timemaster_node" pkg="icarus_rover_v2" type="timemaster_node"  output="screen" clear_params="true">
	    <param name="verbosity_level"       value="DEBUG"/> 
        <param name="pps_source"            value="self"/>
        <param name="require_pps_to_start"  value="false"/>
        <param name="loop1_rate"            value="1"/> <!-- /1PPS -->
        <param name="loop2_rate"            value="0.1"/> <!-- /01PPS -->
        <param name="loop3_rate"            value="5"/>
    </node>

</launch>
